build.sbt

```diff
@@ -126,15 +118,6 @@ def playProject(label: String, projectName: String, domainPrefix: String, devHtt
         s"-J-Xloggc:/var/log/${packageName.value}/gc.log"
       ),
       commonSettings,
-      initialize := {
-        val _ = initialize.value
-        val expectedJavaVersion = parseJavaVersionNumber(javaVersionNumber)
-        val actualJavaVersion = removeStartingOne(sys.props("java.specification.version"))
-        assert(
-          expectedJavaVersion.equals(actualJavaVersion),
-          s"Java $expectedJavaVersion is required for this project. You are using Java $actualJavaVersion."
-        )
-      },
```

Comment:
We now require that projects be able to _build_ with Java 21 if they're using Scala Steward, even if they aren't _using_ Java 21, so this strict rule here of requiring exactly Java 11 for the build is not helpful. We _could_ have changed it to a greater-than-or-equals condition, but that's already accomplished by the `-release:11` config already present:

https://github.com/guardian/typerighter/blob/b12225dc7eafe1b2b3ed6e4c1103ed9fccc74c49/build.sbt#L11